pyclient/deephaven/session.py

```diff
@@ -0,0 +1,279 @@
+import os
+import threading
+from concurrent.futures import ThreadPoolExecutor
+
+from bitstring import BitArray
+
+from deephaven.arrow_flight_service import ArrowFlightService
+from deephaven.console_service import ConsoleService
+from deephaven.dherror import DHError
+from deephaven.proto import barrage_pb2_grpc, ticket_pb2
+from deephaven.query import Query
+from deephaven.session_service import SessionService
+from deephaven.table_service import TableService
+
+
+class Session:
+    def __init__(self, host='localhost', port=10000, user="", password=""):
+        self._r_lock = threading.RLock()
+        self._last_ticket = 0
+        self._ticket_bitarray = BitArray(1024)
+        self.host = os.environ.get("DHCE_HOST", host)
+        self.port = os.environ.get("DHCE_PORT", port)
```

Comment:
Typically, we want an explicitly passed in value to take precedence over an environment variable, right?

I think we want the precedence:

1. Manually value
2. Env value
3. Default value